src/util/MetadataTypes.ts

```diff
@@ -0,0 +1,9 @@
+import { namedNode } from '@rdfjs/data-model';
+import { NamedNode } from 'rdf-js';
+
+export const TYPE: NamedNode = namedNode('http://www.w3.org/1999/02/22-rdf-syntax-ns#type');
+export const CONTENT_TYPE: NamedNode = namedNode('http://www.w3.org/ns/ma-ont#format');
+export const SLUG: NamedNode = namedNode('http://example.com/slug');
```

Comment:
Can we use an existing vocab for this and the following?

Reply:
I looked into which ontologies I could find that match those things, hence the content type predicate, but couldn't immediately find anything for the others that can easily be used. There is the HTTP ontology, but there you have like 5 links between the identifier and the actual header value.

(I actually forgot I put example predicates there, was also going to be one of the discussion points on what to do there 😅 )

Reply:
We can always just defer by opening an issue if no clear candidates exist :-)

Reply:
Let's not use `example.com` but rather an internal URN or something, for instance `urn:solid:`.